modules/core/src/main/java/org/apache/ignite/internal/commandline/DeactivateCommand.java

```diff
@@ -78,6 +81,21 @@
         return null;
     }
 
+    /** {@inheritDoc} */
+    @Override public void parseArguments(CommandArgIterator argIter) {
+        force = false;
```

Comment:
Do we need this line? Default `force` value is `false` already.

Reply:
> Do we need this line? Default `force` value is `false` already.

Yes, we need this. Commands are statefull unfortunately. Without this, next command usage will go with previous value 'force' param.